osu.Game/Beatmaps/BeatmapInfo.cs

```diff
@@ -95,6 +94,9 @@ public class BeatmapInfo : IEquatable<BeatmapInfo>, IJsonSerializable, IHasPrima
         public bool WidescreenStoryboard { get; set; }
         public bool EpilepsyWarning { get; set; }
 
+        public CountdownType Countdown { get; set; } = CountdownType.Normal;
+        public int CountdownOffset { get; set; }
```

Comment:
We should probably start xmldoc'ing these. I have no idea what this means, and there doesn't seem to be a usage of it yet which makes it hard to infer meaning. I'm guessing this is the number of... beats to offset the countdown by? Not sure on directionality though.

Reply:
Correct, this setting moves the start of the countdown N beats back. I'll add xmldoc to these if they don't end up slated for removal from the db model after all (as per the other comment).